UKHomeOffice/sgmr-service
# NMSW-1170 Set support email as a local variable

### AC / Description of changes

Given I am a public user of the SPCR website
When I am shown an email address to use to request support
Then it should be set to the support email address

Local variable in the FE code set to the support email address
That variable used anywhere the email address should be visible in the UI
Remove any instances of `GOV_NOTIFY_SUPPORT_EMAIL` as that will not be used for the customer facing email address 

### To Test

- run app
- the app should run
- check that email is visible on the help footer page and the accessibility statement (cannot find any other pages that use it in the app)
- check the code that all instances of `GOV_NOTIFY_SUPPORT_EMAIL` have been removed

### Notes

---

**Remember to merge to feature branches not `master`.**
